bot/exts/help_channels/_cog.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, bot: Bot):
         self.bot = bot
         self.scheduler = scheduling.Scheduler(self.__class__.__name__)
         self.help_forum_channel: discord.ForumChannel = None
+        self.check_all_open_posts_have_close_task.start()
```

Comment:
Should this be at the end of `cog_load` to ensure `self.help_forum_channel` is set first?

Reply:
Yea good point, lets do that just to be safe. Updated.